cluster/images/hyperkube/addons/dashboard-rc.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
```

Comment:
Can you add a comment here that this files should be kept in sync with cluster/addons/dashboard/dashboard-controller.yaml? And update comment there to also stay in sync with this one.